src/EnergyPlus/DataGlobals.hh

```diff
@@ -65,7 +66,7 @@ namespace DataGlobals {
 
     extern bool runReadVars;
     extern bool DDOnlySimulation;
-    extern bool AnnualSimulation;
+    //!$extern bool AnnualSimulation;
```

Comment:
Are are these commented out instead of just removed if they are unused?

Reply:
just left over from prototyping.  they will get removed